.github/workflows/build_and_test.yml

```diff
@@ -11,7 +11,6 @@ env:
   TURBO_VERSION: 2.3.3
   NODE_MAINTENANCE_VERSION: 18
   NODE_LTS_VERSION: 20
-  TEST_CONCURRENCY: 8
```

Comment:
Removed from `build_and_test` because this isn't actually doing anything. It's the environment variable in `build_reusable` that's used. This was potentially misleading.